app/src/main/java/app/revanced/integrations/patches/SpoofSignatureVerificationPatch.java

```diff
@@ -76,62 +76,91 @@ public static void onResponse(int responseCode) {
     /**
      * Last WindowsSetting constructor values. Values are checked for changes to reduce log spam.
      */
-    private static int lastAnchorPosition, lastAnchorHorizontal, lastAnchorVertical;
+    private static int lastAp, lastAh, lastAv;
     private static boolean lastVs, lastSd;
 
+    /**
+     * Known incorrect default Shorts subtitle parameters, and the corresponding (non-Shorts) correct values.
+     */
+    private static final SubtitleWindowReplacementSettings[] knownReplacements = {
+            new SubtitleWindowReplacementSettings(9, 20, 0, true, false,
+                    34, 50, 90),
+            new SubtitleWindowReplacementSettings(10, 50, 0, true, false,
+                    34, 50, 95),
+            new SubtitleWindowReplacementSettings(9, 20, 0, true, true,
+                    33, 20, 100),
+    };
```

Comment:
At this point probably an enum is a better choice.

Reply:
Can a name be inferred from the values? It is known what those values do, maybe a name can be inferred from that for each combination of values.

Reply:
Some name along these lines sounds fine to me.